include/asterisk/config.h

```diff
@@ -59,6 +59,18 @@ enum config_save_flags {
 #define	CONFIG_STATUS_FILEUNCHANGED	(void *)-1
 #define	CONFIG_STATUS_FILEINVALID	(void *)-2
 
+/*!
+ * \brief Types used for ast_include_new() include_type
+ */
+typedef enum {
+	/*! #include statements */
+	CONFIG_STATEMENT_INCLUDE    = 0,
+	/*! #exec statements */
+	CONFIG_STATEMENT_EXEC       = 1,
+	/*! #tryinclude statements */
+	CONFIG_STATEMENT_TRYINCLUDE = 2,
+} include_statement_type;
```

Comment:
I think my previous comments were confusing.

You should add this `enum` only in `main/config.c` - `ast_include_new()` will still be forced to take an `int` for API compatibility.

Reply:
Understood.  I changed the `ast_include_new()` argument type back to an `int` and moved the `enum` to `main/config.c`.